easybuild/easyconfigs/m/MACS2/MACS2-2.2.7.1-foss-2021a.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonPackage'
+
+name = 'MACS2'
+version = '2.2.7.1'
+
+homepage = 'https://github.com/taoliu/MACS'
+description = "Model Based Analysis for ChIP-Seq data"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['ad2ca69bdd02a8942a68aae23133289b5c16ba382bcbe20c39fabf3948929de5']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Boost', '1.76.0'),
```

Comment:
Previous easyconfigs do not have `Boost` as a dependency, but do have `SciPy-bundle` (to provide `numpy`). [The failure](https://github.com/easybuilders/easybuild-easyconfigs/pull/18370#issuecomment-1712755160) confirms the need for `numpy`:
```
== 2023-09-10 08:48:02,050 build_log.py:171 ERROR EasyBuild crashed with an error (at easybuild/easybuild-framework/easybuild/base/exceptions.py:126 in __init__): Sanity check failed: found one or more downloaded dependencies: Collecting numpy>=1.17 (at easybuild/easybuild-framework/easybuild/framework/easyblock.py:3655 in _sanity_check_step)
```